includes/llms.template.functions.php

```diff
@@ -261,19 +261,6 @@ function lifterlms_template_single_course_tracks() {
 	}
 }
 
-/**
```

Comment:
Why was this code removed? If it was an accident, add it back. If not, let's include an explanation in the PR description and something about it in the changelog.

Reply:
Ah, the file was edited at one point, and the CS fixer removed the extra line. Since there's no other tangible changes I'll just revert the change to the file to avoid confusion.

Reply:
The function is in the same file twice (it's also on line 245 above). Thanks to the `function_exists()` check there was no error, but didn't need to be defined a second time. It still exists so no changelog needed :)